elliottlib/attach_cve_flaws.py

```diff
@@ -37,24 +34,37 @@ def get_corresponding_flaw_bugs(bzapi, tracker_bugs):
 
 
 def is_first_fix(bzapi, flaw_bug, current_target_release, tracker_ids_to_be_ignored=[]):
-    other_flaw_trackers = bzapi.query(bzapi.build_query(
+    """
+    Check if a flaw bug is considered a first-fix for a target release
+    """
+    # get all the tracker bugs for a flaw bug
+    # but only for OCP product
+    tracker_ids = [t for t in flaw_bug.depends_on if t not in tracker_ids_to_be_ignored]
+    tracker_bugs = bzapi.query(bzapi.build_query(
```

Comment:
I think you need to only be comparing against trackers for the same component.

Reply:
This will be fixed in upcoming story.